skcosmo/datasets/descr/nice_dataset.rst

```diff
@@ -0,0 +1,61 @@
+.. _nice-dataset:
+
+NICE dataset
+##########
+This is a toy dataset containing NICE[1, 4] features for first 500 configurations of the dataset[2, 3] with randomly displaced methane configurations. 
```

Comment:
Can you define the acronym NICE somewhere here?

Reply:
Updated. Thanks for the comment!